ops/docker/ci-builder/Dockerfile

```diff
@@ -54,14 +54,9 @@ RUN apt-get update && \
   pip install slither-analyzer==0.9.3 && \
   go install gotest.tools/gotestsum@latest && \
   curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.48.0 && \
-  curl -fLSs https://raw.githubusercontent.com/CircleCI-Public/circleci-cli/master/install.sh | bash &&
-  chmox +x /usr/local/bin/check-changed
```

Comment:
did you mean to get rid of check changed chmod here?

Reply:
yeah. This was removed before by https://github.com/ethereum-optimism/optimism/pull/6065. But got readded due to a merge conflict.